### DISCUSSION TITLE
Project setup issue, not able to run

### DISCUSSION THREAD

**Discussion by Najaf93** [google/model-viewer#4189] [Q&A]
![image](https://user-images.githubusercontent.com/6262806/229049901-0a24770d-aca3-438f-b713-c798bf70dc8f.png)

Hi,
 I am getting this error when I enter "npm run bootstrap" command. Also, i followed all the WSL2 installation and setup steps.
Anyone, please help me figure out the issue here?
Thanks in advance. Cheers!

**Comment by elalish** [comment]
Did you remember `npm install`?
